noir_stdlib/src/lib.nr

```diff
@@ -60,6 +60,5 @@ pub fn wrapping_mul<T>(x : T, y: T) -> T {
     crate::from_field(crate::as_field(x) * crate::as_field(y))
 }
 
-pub fn wrapping_shift_left<T>(x : T, y: T) -> T {
-    crate::from_field(crate::as_field(x) * 2.pow_32(crate::as_field(y)))
-}
```

Comment:
Add doc comment for what this is doing please

Reply:
Done